sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -716,7 +716,6 @@ public void processElement(ProcessContext c) throws Exception {
       public void finishBundle() throws Exception {
         bigtableWriter.flush();
         checkForFailures();
-        LOG.info("Wrote {} records", recordsWritten);
```

Comment:
Consider swapping this to debug instead of dropping the logging statement completely.